### PR TITLE
link to micropython-usermod

### PR DESCRIPTION
**Pull request by dholth**
This is an online book/documentation but also strongly related to the nearby links. Where would you put it?

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -920,6 +920,7 @@ input via push buttons or a navigation joystick and an optional rotary encoder.
 * [micropy-stubs](https://github.com/BradenM/micropy-stubs) - Automatically Generated Stub Packages for Micropy-Cli and whomever else.
 * [micropython-extmod-generator](https://github.com/prusnak/micropython-extmod-generator) - Generator for MicroPython external modules written in C.
 * [micropython-package-template](https://github.com/brainelectronics/micropython-package-template) - GitHub workflow supported MicroPython package template with deploys to the [Python Package Index](https://pypi.org/) on a push to the main branch and test deploys to the [Test Python Package Index](https://test.pypi.org/) on PRs.
+* [micropython-usermod](https://micropython-usermod.readthedocs.io) - Online book about MicroPython external modulules writen in C.
```
Comment (andypiper) [inline readme.md:923]: typo "modulules" -> "modules"

**Comment by andypiper** [comment]
I think you've chosen a good spot, personally, but I can see why it could also fit into documentation.

File: readme.md (hunk 1 above)
Comment (dholth) [reply readme.md:923]: modukeleles

**Comment by dholth** [comment]
Fixed.

**Comment by mcauser** [comment]
Thanks @dholth

**Pull request merged**